### PR TITLE
vacation2020

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Comment by oysteinmoseng** [comment]
Note that we will presumably have to go through the PRs and manually add the info to the changelog.

File: ts/Core/Axis/Axis.ts [hunk 1]
```diff
@@ -4121,7 +4120,11 @@ class Axis implements AxisComposition, AxisLike {
         axis.plotLinesAndBandsGroups = {};
 
         // Shorthand types
-        axis.positiveValuesOnly = !!(axis.logarithmic && !options.allowNegativeLog);
+        axis.positiveValuesOnly = !!(
```
Comment (TorsteinHonsi) [inline ts/Core/Axis/Axis.ts:4123]: Please revert this and the other related changes, see https://github.com/highcharts/highcharts/issues/13914#issuecomment-671298850 . @bre1470 .

File: ts/Core/Axis/LogarithmicAxis.ts [hunk 2]
```diff
@@ -204,10 +209,26 @@ class LogarithmicAxisAdditions {
     }
 
     public lin2log(num: number): number {
+        if (this.axis.options.allowNegativeLog) {
```
Comment (TorsteinHonsi) [inline ts/Core/Axis/LogarithmicAxis.ts:212]: I think this should be reverted, see https://github.com/highcharts/highcharts/issues/13914#issuecomment-671298850 . @bre1470 .

File: ts/Core/Axis/GridAxis.ts [hunk 3]
```diff
@@ -105,6 +105,43 @@ var applyGridOptions = function applyGridOptions(axis: Highcharts.Axis): void {
...
 };
 
+/**
+ * For a datetime axis, the scale will automatically adjust to the
+ * appropriate unit. This member gives the default string
+ * representations used for each unit. For intermediate values,
+ * different units may be used, for example the `day` unit can be used
+ * on midnight and `hour` unit be used for intermediate values on the
+ * same axis.
+ * For Gantt possible to declare as a list to provide different
```
[the remark this answers is not in the document]
Comment (karolkolodziej) [reply ts/Core/Axis/GridAxis.ts:115]: Changed in #14022 ;)

**Pull request merged**